app/Core/Router.php

```diff
@@ -29,4 +51,74 @@ private function __construct() {
     public static function getInstance() : Router {
         return self::$routerInstance ?? new Router();
     }
+
+    /**
+     *  Get Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function get($url, $action){
+        $this->addRoute(GET, $url, $action);
+    }   
+
+    /**
+     *  Post Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function post($url, $action){
+        $this->addRoute(POST, $url, $action);
+    }   
+
+
+    /**
+     *  Delete Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function delete($url, $action){
+        $this->addRoute(DELETE, $url, $action);
+    }   
+
+    /**
+     *  Patch Request on url and action
+     *
+     * @param string url
+     * @param string action
+     */
+    public function patch($url, $action){
+        $this->addRoute(PATCH, $url, $action);
+    }
+
+    /**
+     * Adds newly registered route to routes array
+     *
+     * @param string $request_type
+     * @param string url
+     * @param string action
+     */
+    protected function addRoute($request_type='Get', $url, $action){
```

Comment:
Does this line work? I mean PHP doesn't allow default parameters as the first parameter?
Which brings me to the question, have you tested this class to work?